build.gradle

```diff
@@ -65,7 +68,7 @@ grailsPublish {
         puneetbehl: 'Puneet Behl',
         sbglasius: 'Søren Berg Glasius',
         matrei: 'Mattias Reichel',
-        jdaugherty: 'James Daugherty',
+        jdaugherty: 'James Daugherty'
```

Comment:
If you keep the comma it's one left line diff if we ever add someone else.

Reply:
Aha! Well, I'd rather have beautiful code :tulip: